tests/hlsl-intrinsic/bit-cast-double.slang

```diff
@@ -5,19 +5,37 @@
 //TEST(compute):COMPARE_COMPUTE_EX:-slang -cuda -compute -shaderobj
 //DISABLE_TEST(compute):COMPARE_COMPUTE:-slang -shaderobj -mtl
 
-//TEST_INPUT:ubuffer(data=[0 0 0 0], stride=4):out,name=outputBuffer
-RWStructuredBuffer<int> outputBuffer;
+#define THREAD_SIZE 3
 
-[numthreads(4, 1, 1)]
+//TEST_INPUT:ubuffer(data=[0 0 0 0 0 0 0 0 0], stride=4):out,name=outputBuffer1
+RWStructuredBuffer<int> outputBuffer1;
+
+//TEST_INPUT:ubuffer(data=[0 0 0 0 0 0 0 0 0], stride=4):out,name=outputBuffer2
+RWStructuredBuffer<int> outputBuffer2;
```

Comment:
The test here is testing `asdouble`, we should make this function run with a dispatch of (1,1,1) so we only need 1 output buffer (and to simplify the test):

then we will be able to put the result of `double asdouble()` in index 0, and `double2 asdouble()` in index 1